aws/cloudformation/s3_buckets.yml

```diff
@@ -38,3 +38,43 @@ Resources:
         BlockPublicPolicy: true
         IgnorePublicAcls: true
         RestrictPublicBuckets: true
+  # for curriculumbuilder
+  CurriculumBucket:
+    Type: 'AWS::S3::Bucket'
+    Properties:
+      BucketName: 'cdo-curriculum-devel'
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: 'aws:kms'
+      VersioningConfiguration:
+        Status: 'Enabled'
+      LoggingConfiguration:
+        DestinationBucketName: 'cdo-logs'
+        LogFilePrefix: 's3/cdo-curriculum-devel'
+      PolicyDocument: {
+        "Version": "2008-10-17",
+        "Statement": [
+            {
+                "Sid": "PublicReadForGetBucketObjects",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "*"
+                },
+                "Action": "s3:GetObject",
+                "Resource": "arn:aws:s3:::cdo-curriculum-devel/*"
+            },
+            {
+                "Sid": "",
+                "Effect": "Allow",
+                "Principal": {
+                    "AWS": "arn:aws:iam::475661607190:user/josh@code.org"
```

Comment:
Is this the IAM user that's used for actual curriculumbuilder? Let's use a different one for the development bucket since presumably devs will have it stored in a local file. Also, since Josh doesn't work here anymore, we should probably be moving away from using that credential in general, not using it in more places.

Reply:
+1

Reply:
I'll go ahead and make a new IAM user

Reply:
I just remembered that we want to move away from IAM users. You should be able to use the temporary credentials that are saved locally when you authenticate with Google to AWS. (This is the process we use for all AWS access through the CLI and from the main application locally).

If you can configure CB to use the `cdo` profile, that should work.

Reply:
Synced offline -
The new plan is to create an IAM role with scoped down permissions that we will assume